asv/plugins/virtualenv.py

```diff
@@ -108,24 +97,30 @@ def name(self):
 
     @classmethod
     def matches(self, python):
-        if not (re.match(r'^[0-9].*$', python) or re.match(r'^pypy[0-9.]*$', python)):
+        match = re.match(r'\A(?:pypy)?([0-9.]*)\Z', python)
+        if not match:
             # The python name should be a version number, or pypy+number
             return False
+        executable = Virtualenv._find_python(python)
 
-        try:
-            import virtualenv
-        except ImportError:
-            return False
+        if LooseVersion(match.group(1)) < LooseVersion("3.3"):
+            self._virtualenv_argv = [
+                sys.executable, "-mvirtualenv", "--no-site-packages"]
```

Comment:
Even if I rename "self" to "cls" everywhere in the class (which I'm happy to do) it should not change the semantics of the method (basically even now I'm just setting the attribute on the class rather than on the attribute, which should not matter).

I cannot reproduce the failure of test_run_build_failure or of test_run_spec locally (looks like pip is not getting installed for a reason I can't fathom), they pass for me locally.  (test_web_regressions needs the selenium test driver which I don't have installed but given that it's essentially the same failure I don't think it's relevant).